app/src/main/res/menu/mediaplayer.xml

```diff
@@ -63,27 +63,5 @@
         android:menuCategory="container"
         custom:showAsAction="never"
         android:title="@string/share_label">
```

Comment:
I would use `share_label_with_ellipses` here

Reply:
yup, adde it